extraction/data_models/contents.py

```diff
@@ -0,0 +1,60 @@
+"""This script creates object structure to store the extracted data
+    and provides methods to set and retrieve the attributes"""
+
+import enum
+
+
+class Contents:
+    """This class creates content_list which is list of extracted contents"""
+    def __init__(self):
+        self.content_list = list()
+
+    def add_content(self, _content):
+        """This method appends extracted content to content_list"""
+        self.content_list.append(_content)
+
+    def __str__(self):
+        return_string = '[\n'
+
+        for content in self.content_list:
+            return_string = '{}{}\n\t}},\n'.format(return_string, str(content))
+
+        return return_string + ']'
+
+
+class _Content:
+    """This class creates object representation for each extracted content"""
+
+    counter = 0
```

Comment:
Let's make this part of Contents - for now this could be the index of the element in the list. 

We wouldn't need the index and get_index methods. Since the content will be accessed from Contents class above, we shouldn't need this here.

(With the current implementation, you wouldn't be able to work on more than one page since counter is static to the class. Also if 2 elements are on the same level, you might later on need to give them the same index - this should be a problem Contents should handle)

Reply:
Thanks for suggestion! Updated.